specification/translation/Azure.AI.DocumentTranslation/routes.tsp

```diff
@@ -0,0 +1,435 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Versioning;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace DocumentTranslation;
+
+alias ServiceTraits = SupportsRepeatableRequests &
+  SupportsConditionalRequests &
+  SupportsClientRequestId;
+
+/** API to translate a document. */
+#suppress "@azure-tools/typespec-azure-core/byos"
+@post
+@route("document:translate")
+op documentTranslate is RpcOperation<
```

Comment:
Why is this operation outside of the interface?

Reply:
Moved it inside the interface